specification/sql/resource-manager/Microsoft.Sql/preview/2020-08-01-preview/ServerDevOpsAudit.json

```diff
@@ -16,13 +16,13 @@
     "application/json"
   ],
   "paths": {
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/devOpsAuditingSettings/{devOpsAuditPolicyName}": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/devOpsAuditingSettings/{devOpsAuditingSettingsName}": {
```

Comment:
can we keep previous name? as it will cause sdk breaking change

Reply:
This is a new swagger, SDK is not published yet.